cpp/src/svm/svc.h

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Provide `svm_api.h` and `svm_api.cpp` to extern the stateless API functions through a C API.

Reply:
C wrappers added.